requirements.txt

```diff
@@ -8,6 +8,10 @@ torch
 num2words==0.5.10
 word2number==1.1
 
+# for SuspectingParaphraser
+pyinflect==0.5.1
+gender-guesser==0.4.0
+
```

Comment:
Hi @wwydmanski, it would be necessary to look for a substitute library to gender-guesser which uses MIT/Apache 2 or a similar license to merge this PR.

Reply:
Will do, thanks for letting me know